controllers/argocd/statefulset.go

```diff
@@ -591,3 +598,31 @@ func updateNodePlacementStateful(existing *appsv1.StatefulSet, ss *appsv1.Statef
 		*changed = true
 	}
 }
+
+//to fetch the broken pod of StatefulSet if any, this function intends to delete the broken pod that cannot
```

Comment:
Please correct the comments as per the standards
https://go.dev/blog/godoc

I would suggest something like this.
// Returns true if a StatefulSet has  pods in ErrImagePull or ImagePullBackoff state.

Reply:
thanks, will update